scripts/bump/exec.sh

```diff
@@ -14,14 +14,15 @@ alias ghr="curl https://api.github.com/repos/$REPO/branches/$branch/protection \
   -s"
 
 # Updating the registration version for all packages
-find "$packages_dir"/*/src/index.ts -exec sed -i "s/version:.\+\$/version: \/* updated-by-script *\/ \'$VERSION_TAG\',/m" {} +
+find "$packages_dir"/*/src/index.ts -exec sed -i'' -e "s/version:.\+\,/version: \/* updated-by-script *\/ \'$VERSION_TAG\',/" {} +
 
 npx lerna version "$VERSION_TAG" --no-git-tag-version --no-push --yes
 
 # Updating the peer dependencies in packages
 find "$packages_dir"/*/package.json -exec node "$bump_scripts_dir"/package-update.js -v "$VERSION_TAG" {} +
 
 # Updating package-lock.json to reflect results of the previous command
+npx lerna clean --yes
```

Comment:
Cross-linked packages were not updated in `package-lock.json` if they are “installed”. Removing the install makes the next step update `package-lock.json` for the new versions, as intended.